src/platforms/windows/daemon/wireguardutilswindows.cpp

```diff
@@ -421,7 +430,7 @@ QList<WireguardUtils::PeerStatus> WireguardUtilsWindows::getPeerStatus() {
   if (!m_adapter) {
     return {};
   }
-  DWORD bufferSize = 1024;
+  DWORD bufferSize = 2048;
   auto buffer = std::array<uint8_t, 2048>{};
```

Comment:
the size was not the size of the buffer, not causing bugs (given we announce less space), that should be the same.